CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 ### Changed
 
 - Replace `ExpressibleByStringLiteral` with `ExpressibleByStringInterpolation` for `ProjectDescription` objects by [@DimaMishchenko](https://github.com/DimaMishchenko)
+- Fix adding framework targets to AppClip by [@sampettersson](https://github.com/sampettersson)
```

Comment:
add a link to the PR please (see below) => I think the log above does not have a PR link by mistake